handlers/mempoolView.go

```diff
@@ -47,7 +45,7 @@ func formatToTable(content *types.RawMempoolResponse) *types.DataTableResponse {
 		for _, tx := range pendingData {
 			dataTable.Data = append(dataTable.Data, []any{
 				// TODO: link to tx page once it has a mempool view implemented
-				template.HTML(fmt.Sprintf(`<span class="text-monospace" >%v</span>`, tx.Hash.String())),
+				utils.FormatAddressWithLimits(tx.Hash.Bytes(), "", false, "tx", 15, 18, true),
```

Comment:
A hash is not an address, using this format function seems dangerous. I guess you took the code from
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/ff0f06795870e1258145310e6e3cab58de2d56f8/handlers/eth1Block.go#L160

I will talk to the according dev, we might want to change that.

This will most likely happen after this PR, feel free to resolve this comment, it is more FYI.